apps/mdn/mdn-aws/k8s/Makefile

```diff
@@ -178,6 +178,23 @@ export ELB_S3_LOGGING_BUCKET ?= mdn-elb-logs
 # ELB_S3_LOGGING_PREFIX can't end with a slash!
 export ELB_S3_LOGGING_PREFIX ?= ${TARGET_ENVIRONMENT}
 
+### New Relic configuration
+export NEW_RELIC_SECRETS_NAME ?= newrelic-secrets
+export NEW_RELIC_LICENSE_KEY ?= foo
```

Comment:
I think this line is not needed?

Reply:
it is, I forgot the make targets that use these.